x-pack/plugins/canvas/server/collectors/workpad_collector.ts

```diff
@@ -151,11 +165,23 @@ export function summarizeWorkpads(workpadDocs: CanvasWorkpad[]): WorkpadTelemetr
         }
       : undefined;
 
+  const countByTemplateName = templates.reduce((accum: Record<string, number>, t: string) => {
+    accum[t] = accum[t] ? accum[t] + 1 : 1;
+
+    return accum;
+  }, {});
+
+  const templatesInfo = {
+    total: templates.length,
+    by_name: countByTemplateName,
+  };
+
   return {
     workpads: { total: workpadsInfo.length },
     pages: pagesInfo,
     elements: elementsInfo,
     functions: functionsInfo,
+    templates: templatesInfo,
```

Comment:
Maybe I should rename this to be templateUsage? Not to be confused with something related to number of templates installed or something

Reply:
Maybe templatesUsage sounds more forward looking, yes.  We can add templatesCreated and other functionality soon, as the goal is to have a way for people to upload their templates to the Package Registry in the future. 👍